src/main/java/mate/academy/hibernate/relations/exception/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.hibernate.relations.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message) {
+        super(message);
```

Comment:
In my opinion, it's better to have a constructor with two parameters: message and exception object